plugins/inputs/vsphere/README.md

```diff
@@ -551,9 +561,231 @@ For a detailed list of commonly available metrics, please refer to
+  * virtual-machine
+    * iops_read, throughput_read, latency_read_avg, latency_read_stddev, read_count, iops_write, throughput_write, latency_write_avg, latency_write_stddev, write_count
+  * vscsi
+    * iops_read, throughput_read, latency_read, read_count, iops_write, throughput_write, latency_write, write_count
+  * virtual-disk
+    * iops_limit, niops, niops_delayed
+  * vsan-host-net
+    * rx_throughput, rx_packets, rx_packets_loss_rate, tx_throughput, tx_packets, tx_packets_loss_rate
+  * vsan-vnic-net
+    * rx_throughput, rx_packets, rx_packets_loss_rate, tx_throughput, tx_packets, tx_packets_loss_rate
+  * vsan-pnic-net
+    * rx_throughput, rx_packets, rx_packets_loss_rate, tx_throughput, tx_packets, tx_packets_loss_rate
+  * vsan-iscsi-host
+    * iops_read, iops_write, iops_total, bandwidth_read, bandwidth_write, bandwidth_total, latency_read, latency_write, latency_total, queue_depth
+  * vsan-iscsi-target
+    * iops_read, iops_write, iops_total, bandwidth_read, bandwidth_write, bandwidth_total, latency_read, latency_write, latency_total, queue_depth
+  * vsan-iscsi-lun
+    * iops_read, iops_write, iops_total, bandwidth_read, bandwidth_write, bandwidth_total, latency_read, latency_write, latency_total, queue_depth
+
+### vSAN Tags
+
+* all vSAN metrics
+  * vcenter
+  * dcname
+  * clustername
+  * moid (the cluster's managed object id)
+* host-domclient, host-domcompmgr
+  * hostname
+* disk-group, cache-disk, capacity-disk
+  * hostname
+  * deviceName
+  * ssdUuid (if SSD)
+* vsan-host-net
+  * hostname
+* vsan-pnic-net
+  * pnic
+* vsan-vnic-net
+  * vnic
+  * stackName
+
+### Realtime vs. Historical Metrics in vSAN
+
+vSAN metrics also keep two different kinds of metrics - realtime and
+historical metrics.
+
+* Realtime metrics are metrics with the prefix 'summary'. These metrics are available in realtime.
+* Historical metrics are metrics with the prefix 'performance'. These are metrics queried from vSAN performance API, which is available at a 5-minute rollup level.
+
+For performance consideration, it is better to specify two instances of the
+plugin, one for the realtime metrics with a short collection interval,
+and the second one - for the historical metrics with a longer interval.
+For example:
+
+```toml
+## Realtime instance
+[[inputs.vsphere]]
+  interval = "30s"
+  vcenters = [ "https://someaddress/sdk" ]
+  username = "someuser@vsphere.local"
+  password = "secret"
+
+  insecure_skip_verify = true
+  force_discover_on_init = true
+
+  # Exclude all other metrics
+  vm_metric_exclude = ["*"]
+  datastore_metric_exclude = ["*"]
+  datacenter_metric_exclude = ["*"]
+  host_metric_exclude = ["*"]
+  cluster_metric_exclude = ["*"]
+  
+  vsan_metric_include = [ "summary.*" ]
+  vsan_metric_skip_verify = false
+
```

Comment:
exclude is still `*` (default)?

Reply:
This may clarify your doubts - [performance-considerations](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/vsphere#performance-considerations)

Reply:
Please update config so it is correct, just like on line 640.